### PR TITLE
Increase stack ulimit to 8Mb to address build failures for gcc with long command lines

### PR DESCRIPTION
**Pull request by yanavlasov**
Signed-off-by: Yan Avlasov <yavlasov@google.com>

### CODE REVIEW

File: toolchains/rbe_toolchains_config.bzl [hunk 1]
```diff
@@ -96,7 +96,7 @@ def _envoy_rbe_toolchain(name, env, toolchain_config_spec_name, toolchain_config
     )
 
 def rbe_toolchains_config(generator = False, force = False):
-    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard")
+    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard", docker_ulimits="memlock=-1,nice=-20,rtprio=10,stack=8192")
```
Comment (phlax) [inline toolchains/rbe_toolchains_config.bzl:99]: i think you might need to `*1024` - not sure how rbe works but testing locally - eg:

```
$ docker run -ti --rm --ulimit stack=8388608 busybox sh
/ # ulimit -s
8192
```

Comment (yanavlasov) [reply toolchains/rbe_toolchains_config.bzl:99]: Yes, good catch

**Review by phlax — APPROVED** [review]
it would be nice if we could test it - but probs easier to just land

lgtm, thanks

**Comment by yanavlasov** [comment]
It is easier to test with it submitted. We can always revert.

File: toolchains/rbe_toolchains_config.bzl [hunk 2]
```diff
@@ -96,7 +96,7 @@ def _envoy_rbe_toolchain(name, env, toolchain_config_spec_name, toolchain_config
     )
 
 def rbe_toolchains_config(generator = False, force = False):
-    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard")
+    linux_exec_properties = create_rbe_exec_properties_dict(docker_add_capabilities = "SYS_PTRACE,NET_RAW,NET_ADMIN", docker_network = "standard", docker_ulimits="memlock=-1,nice=-20,rtprio=10,stack=8388608")
```
Comment (antoniovicente) [inline toolchains/rbe_toolchains_config.bzl:99]: What is the current stack config for the RBE toolchain on docker?  I think I have been able to repo the long argument list problem when building with gcc locally despite runs of `ulimit -s` on my machine returning 8192

**Pull request merged**